src/ecs-mcp-server/README.md

```diff
@@ -27,7 +27,50 @@ Customers can list and view their ECS resources (clusters, services, tasks, task
 
 ## Installation
 
-### Prerequisites
+### Option 1 (Recommended): Hosted MCP Server
+
+Use the AWS-managed ECS MCP Server for simplified setup and automatic updates. The hosted service eliminates local installation requirements and provides enterprise-grade security through AWS IAM integration.
+
+#### Prerequisites
+
+- [Python 3.10+](https://www.python.org/downloads/release/python-3100/) and [uv package manager](https://docs.astral.sh/uv/getting-started/installation/)
+- [AWS CLI configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) with valid credentials
+- IAM permissions for ECS MCP
+
+#### Configuration
+
+The hosted ECS MCP Server uses the [MCP Proxy for AWS](https://github.com/aws/mcp-proxy-for-aws) to provide secure, authenticated access. Add the following to your MCP client configuration file:
+
+**For Mac/Linux:**
```

Comment:
Lets add windows instructions too?

Or redirect user to the documentation page so we don't have to update the readme here

Reply:
Sounds good